.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  push:
+    paths:
+      - '**/*.py'
+      - 'ui/raidboss/**'
```

Comment:
I think this needs to include all js files as well.  There are tests for things like `resources/regexes.js` and maybe some day we'll have oopsy tests?